api/run.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+import uvicorn
+
+
+if __name__ == "__main__":
+    uvicorn.run(
```

Comment:
The [dev setup](https://www.uvicorn.org/settings/#development) of uvicorn needs the `--reload` flag (or `reload=True`?).

Reply:
Reload is passed on line 11 :slightly_smiling_face:

Reply:
🤦‍♂️ Ah yes. Sorry, I don't know how I did not see it.